### ISSUE TITLE
Check this out

### ISSUE THREAD

**Issue by Leviterminator** [iblobtouch/iblobtouch.github.io#41]
I know i made a lot of dumb requests, but now i would like to see the bullet penetration (of course, after you fix the auto turrets). I say this. because it is really boring sometimes, that when i shoot a big ass bullet, i disappears when it hits a cube, and i made the damage higher, the same thing happens.

AND THE AUTO TURRETS ARE AWESOME, JUST KEEP UP THE HARD WORK, AND THIS SITE WOULD BE THE BEST!!!!!!

**Comment by guardtown** [comment]
Use multiple barrels, with exactly the same settings, but split the total damage to make so all barrels do the amount of damage you want, this way, it gets a semi-penetration.

**Comment by Leviterminator** [comment]
Yeah, i tried that one, but if i want to add a little spread, then this method no longer works, because the bullets will not go to the same direction. And this way, it causes bad performance, because multiple barrels are fired at once.

**Comment by Iblobtouchfan** [comment]
Adding more barrels causes "penetration" but things mess up when spread is added.

**Comment by DogeisCut** [comment]
welp forget it he stoped
